packages/xchain-thorchain/src/client.ts

```diff
@@ -219,14 +219,14 @@ class Client implements ThorchainClient, XChainClient {
 
   // Need to be updated
   getFees = async (): Promise<Fees> => {
```

Comment:
same question here

Reply:
^ Same reason as mentioned in https://github.com/xchainjs/xchainjs-lib/pull/70#discussion_r517481630